app/screens/phone-auth-screen/phone-auth.tsx

```diff
@@ -108,15 +112,19 @@ const styles = EStyleSheet.create({
   }
 })
 
-export const WelcomePhoneInputScreen = ({ navigation }) => {
+type WelcomePhoneInputScreenProps = {
+  navigation: any,
+}
+
+export const WelcomePhoneInputScreen: ScreenType = ({ navigation }: WelcomePhoneInputScreenProps) => {
   const [requestPhoneCode, { loading }] = useMutation(REQUEST_PHONE_CODE, {
     fetchPolicy: "no-cache"
-  });
+  })
 
-  const inputRef = useRef()
+  const inputRef: any = useRef()
```

Comment:
here are some doc to type a mutable useRef the correct way: https://github.com/typescript-cheatsheets/react#option-2-mutable-value-ref